utils_nlp/dataset/pytorch.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import collections
+from torch.utils.data import Dataset
+
+
+class SCDataSet(Dataset):
+    def __init__(self, df, text_col, label_col):
```

Comment:
in the init, we can also add docstrings and they are parsed by sphinx thanks to a command we added https://github.com/microsoft/nlp/blob/master/docs/source/conf.py#L229